test/test_torchinductor_opinfo.py

```diff
@@ -0,0 +1,707 @@
+    "select": {b8},
+    "select_scatter": {b8},
+    "sgn": {b8, i32, i64},
+    "short": {b8, f16, f32, f64, i32, i64},
+    "sign": {b8, i32, i64},
+    "slice_scatter": {b8},
+    "softmax": {f16, f32, f64},
+    "softmax.with_dtype": {b8, f16, f32, f64, i32, i64},
+    "special.polygamma.special_polygamma_n_0": {b8, f16, f32, f64, i32, i64},
+    "split": {b8},
+    "split.list_args": {b8},
+    "split_with_sizes": {b8},
+    "squeeze": {b8, f16, f32, f64, i32, i64},
+    "std_mean": {f16, f32, f64},
+    "stft": {f32, f64},
+    "sum_to_size": {b8, f16, f32, f64, i32, i64},
+    "svd_lowrank": {f32, f64},
+    "symeig": {f32, f64},
+    "tensordot": {f16, f32, f64},
+    "to": {b8, f16, f32, f64, i32, i64},
+    "to_sparse": {b8, f16, f32, f64, i32, i64},
+    "unbind": {b8},
+    "unflatten": {b8, f16, f32, f64, i32, i64},
+    "unfold": {b8, f16, f32, f64, i32, i64},
+    "uniform": {f16, f32, f64},
+    "unique": {b8, f16, f32, f64, i32, i64},
+    "unique_consecutive": {b8, f16, f32, f64, i32, i64},
+    "view": {b8, f16, f32, f64, i32, i64},
+    "view_as": {b8, f16, f32, f64, i32, i64},
+    "view_as_complex": {f16, f32, f64},
+    "where": {b8, f16, f32, f64, i32, i64},
+    "zero_": {b8, f16, f32, f64, i32, i64},
+}
+
+
+class TestInductorOpInfo(TestCase):
+    check_model = check_model
+    check_model_cuda = check_model_cuda
+
+    @onlyNativeDeviceTypes
+    @suppress_warnings
+    @skipCUDAMemoryLeakCheckIf(
+        True
+    )  # inductor kernels failing this test intermittently
+    @_ops(op_db)
+    def test_comprehensive(self, device, dtype, op):
+
+        op_name = op.name
+        if op.variant_test_name:
+            op_name += f".{op.variant_test_name}"
+
+        device_type = torch.device(device).type
+
+        if dtype in inductor_skips[device_type].get(op_name, set()):
+            test_expect = TestExpect.SKIP
+            self.skipTest(f"{op_name} in {dtype} not supported")
+        elif dtype in inductor_expected_failures[device_type].get(op_name, set()):
+            test_expect = TestExpect.XFAILURE
+        else:
+            test_expect = TestExpect.SUCCESS
+
+        func = op.get_op()
+
+        def fn(*args, **kwargs):
+            return func(*args, **kwargs)
+
+        requires_grad = (
+            op.supports_autograd
+            and dtype in op.supported_backward_dtypes(device_type)
+            # TODO: OpInfo really ought to error out for this case, but it's
+            # not exercised in test_ops_gradients atm.  The problem is not
+            # complex32 per-se (which is supported by data movement only ops)
+            # but that when we do backwards we expect other ops like add to work
+            and not dtype == torch.complex32
+        )
+        samples = op.sample_inputs(device, dtype, requires_grad=requires_grad)
+
+        for sample_input in samples:
+            args = [sample_input.input] + list(sample_input.args)
+            # kwargs = sample_input.kwargs
```

Comment:
how come we don't need kwargs?

Reply:
We will need to use this... 
Currently, test_torchinductor's check_model() and check_model_cuda() are not supporting kwargs... Will fix this in the coming PRs.